### PR TITLE
DKC3: Add CHANGELOG.md

### PR DESCRIPTION
**Pull request by PoryGone**
## What is this fixing or adding?
Adds a changelog markdown file to the DKC3 World.

## How was this tested?
I read it, and tossed it into a markdown renderer.

### CODE REVIEW

**Review by nicholassaylor — APPROVED** [review]
Log is nicely formatted and free of errors
LGTM

File: worlds/dkc3/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Donkey Kong Country 3 - Changelog
+
+
+## v1.1
+
+### Features
```
Comment (Exempt-Medic) [inline worlds/dkc3/CHANGELOG.md:6]:
```suggestion
### Features:
```

File: worlds/dkc3/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+- KONGsanity option (Collect all KONG letters in each level for a check)
+- Autosave option
+- Difficulty option
+- MERRY option
+- Handle collected/co-op locations
+
+### Bugfixes:
```
Comment (Exempt-Medic) [inline worlds/dkc3/CHANGELOG.md:14]:
```suggestion
### Bug Fixes:
```

File: worlds/dkc3/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+- Fixed Mekanos softlock
+- Prevent Brothers Bear giving extra Banana Birds
+- Fixed Banana Bird Mother check sending prematurely
+- Fix Logic bug with Krematoa level costs
+
+
+## v1.0
+
+### Features
```
Comment (Exempt-Medic) [inline worlds/dkc3/CHANGELOG.md:24]:
```suggestion
### Features:
```

**Review by Exempt-Medic — CHANGES_REQUESTED** [review]
Minor formatting suggestions to match the other changelogs

**Review by Exempt-Medic — APPROVED** [review]

**Review by ThePhar — APPROVED** [review]

**Pull request merged**